finmanflask/auth.py

```diff
@@ -0,0 +1,62 @@
+from flask import (
+    Blueprint, flash, g, redirect, render_template, request, session, url_for, jsonify, Response, json 
+)
+from finmanflask.db import get_engine
+from sqlalchemy.orm import Session
+from sqlalchemy.exc import IntegrityError
+from werkzeug.security import check_password_hash, generate_password_hash
+
+bp = Blueprint('auth', __name__, url_prefix='/auth')
+
+def validate(field, value):
+    """Validate the field of a given value"""
+    # TODO: implement
```

Comment:
For future reference take a look at [NotImplementedError](https://docs.python.org/3/library/exceptions.html#NotImplementedError). It will be much easier for debugging in larger codebases